openshift/qeb-hwt-template.yaml

```diff
@@ -34,6 +34,10 @@ parameters:
     description: "GitHub App Installation ID"
     displayName: "Installation ID"
     required: true
+  - name: ORIGIN
```

Comment:
I thought we have agreed that the ORIGIN is the HEAD_REPO_URL?

@fridex

Reply:
Yes, it is. But I thought we will still propagate origin (set to `HEAD_REPO_URL`) as that's required in the database to have stacks in a quaryable form agnostic to github/gitlab/...